app/_plugins/drops/plugins/examples/konnect.rb

```diff
@@ -6,11 +6,11 @@ module Plugins
       module Examples
         class Konnect < Base
           URLS = {
-            'consumer' => 'https://{us|eu}.api.konghq.com/v2/runtime-groups/{RUNTIME-GROUP-ID}/core-entities/consumers/{CONSUMER_ID}/plugins',
-            'consumer_group' => 'https://{us|eu}.api.konghq.com/v2/runtime-groups/{RUNTIME-GROUP-ID}/core-entities/consumer_groups/{CONSUMER_GROUP_ID}/plugins',
-            'global' => 'https://{us|eu}.api.konghq.com/v2/runtime-groups/{RUNTIME-GROUP-ID}/core-entities/plugins/',
-            'route' => 'https://{us|eu}.api.konghq.com/v2/runtime-groups/{RUNTIME-GROUP-ID}/core-entities/routes/{ROUTE_ID}/plugins',
-            'service' => 'https://{us|eu}.api.konghq.com/v2/runtime-groups/{RUNTIME-GROUP-ID}/core-entities/services/{SERVICE_ID}/plugins'
+            'consumer' => 'https://{us|eu}.api.konghq.com/v2/control-planes/{CONTROL-PLANE-ID}/core-entities/consumers/{CONSUMER_ID}/plugins',
```

Comment:
```suggestion
            'consumer' => 'https://{us|eu}.api.konghq.com/v2/control-planes/{controlPlaneId}/core-entities/consumers/{CONSUMER_ID}/plugins',
```

Can make it all caps too, but think it should match the spec format.